### PR TITLE
Rename route and add to properties

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

**Comment by robdodson** [comment]
Hm I'm kind of on the fence about these changes just from a pure aesthetic perspective. I prefer the look of things in the previous version. I guess route and params are private but don't know if that is a huge deal because they're at the app level so I liked it better without the underscores.

Similarly, I prefer params to routeData only because I think the word data is a "noise word." It doesn't actually describe what it is. The book Clean Code talks about this and it's always stuck with me. "data" is like saying "stuff", it doesn't mean anything to your audience. I guess I went with params originally to have some continuity with Express and page.js style routers.

**Comment by abdonrd** [comment]
I was inspired by [Pesto](https://github.com/PolymerLabs/app-layout/blob/master/templates/pesto/src/recipe-app.html#L168) (app-layout demo).
But okey! I will remove the underscores.

About the params... I update it to `_dataRoute` because the `carbon-route` attribute is called `data`.

Then. Close this PR and create one that only add the actual variables to attributes?

**Comment by robdodson** [comment]
I'm not even sure if we need the variables in the properties object.

**Comment by abdonrd** [comment]
@robdodson okey! Can you check with the team which is the best practice? :)

File: src/psk-app/psk-app.html [hunk 1]
```diff
@@ -191,9 +191,19 @@ <h4 class="title">Polymer Starter Kit - [[params.page]]</h4>
     Polymer({
       is: 'psk-app',
 
+      properties: {
+        _route: {
+          type: Object
+        },
+
+        _routeData: {
```
Comment (samccone) [inline src/psk-app/psk-app.html]: since this is not private and is used in the templates, using _ seems misleading ... thoughts?

Comment (abdonrd) [inline src/psk-app/psk-app.html]: Yes! Check: https://github.com/PolymerElements/polymer-starter-kit/pull/788#issuecomment-208674908 and https://github.com/PolymerElements/polymer-starter-kit/pull/788#issuecomment-208855399. :)

**Comment by abdonrd** [comment]
@robdodson have you news about https://github.com/PolymerElements/polymer-starter-kit/pull/788#issuecomment-208964666? :)

**Comment by robdodson** [comment]
Just based on looking at other carbon-route projects I decided to leave `route` and rename `params` to `pageData` which matches the `/:page` token

**Comment by robdodson** [comment]
Added in 605d8fc

**Comment by abdonrd** [comment]
@robdodson okey! But I meant about https://github.com/PolymerElements/polymer-starter-kit/pull/788#issuecomment-208951962 and https://github.com/PolymerElements/polymer-starter-kit/pull/788#issuecomment-208964666; We should add the properties?

**Comment by robdodson** [comment]
@abdonrd I think we can leave the properties out. They don't serve much of a purpose in my opinion since we're not trying to expose them as a public API

**Comment by abdonrd** [comment]
@robdodson perfect! 👌

**Pull request closed without merging**